omod/src/main/java/org/openmrs/module/appointmentscheduling/rest/resource/openmrs1_9/DailyAppointmentCountsResource1_9.java

```diff
@@ -0,0 +1,106 @@
+package org.openmrs.module.appointmentscheduling.rest.resource.openmrs1_9;
+
+import org.openmrs.Location;
+import org.openmrs.Provider;
+import org.openmrs.VisitType;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.appointmentscheduling.Appointment;
+import org.openmrs.module.appointmentscheduling.AppointmentType;
+import org.openmrs.module.appointmentscheduling.AppointmentDailyCount;
+import org.openmrs.module.appointmentscheduling.api.AppointmentService;
+import org.openmrs.module.appointmentscheduling.rest.controller.AppointmentRestController;
+import org.openmrs.module.webservices.rest.web.ConversionUtil;
+import org.openmrs.module.webservices.rest.web.RequestContext;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.annotation.Resource;
+import org.openmrs.module.webservices.rest.web.representation.DefaultRepresentation;
+import org.openmrs.module.webservices.rest.web.representation.FullRepresentation;
+import org.openmrs.module.webservices.rest.web.representation.RefRepresentation;
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.api.PageableResult;
+import org.openmrs.module.webservices.rest.web.resource.impl.DataDelegatingCrudResource;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.resource.impl.NeedsPaging;
+import org.openmrs.module.webservices.rest.web.response.ResponseException;
+
+import java.util.Date;
+
+@Resource(name = RestConstants.VERSION_1 + AppointmentRestController.APPOINTMENT_SCHEDULING_REST_NAMESPACE + "/appointmentreport",
+        supportedClass = AppointmentDailyCount.class, supportedOpenmrsVersions = {"1.9.*", "1.10.*", "1.11.*", "1.12.*", "2.0.*", "2.1.*", "2.2.*"})
+public class DailyAppointmentCountsResource1_9 extends DataDelegatingCrudResource<AppointmentDailyCount> {
```

Comment:
This likely isn't a DataDelegatingCrudResource - you can't CRUD an AppointmentDailyCount.  It is just a query.  You should not use this base class, and per above AppointmentDailyCount should not be an OpenmrsObject.

Reply:
My idea was to expose all the appointments grouped by date  and the group counts through a REST call. Kindly suggest  for me the best way of doing this